packages/devtools/src/launcher.ts

```diff
@@ -112,7 +112,9 @@ async function launchChrome (capabilities: ExtendedCapabilities) {
     log.info(`Connect Puppeteer with browser on port ${chrome.port}`)
     const browser = await puppeteer.connect({
         ...chromeOptions,
-        browserURL: `http://localhost:${chrome.port}`
+        browserURL: `http://localhost:${chrome.port}`,
+        //@ts-ignore
```

Comment:
Why do we have a `@ts-ignore` here?

Reply:
Puppeteer broke typings in resent versions. It gives error `Type 'null' is not assignable to type 'Viewport | undefined `
https://github.com/puppeteer/puppeteer/issues/6904
https://github.com/puppeteer/puppeteer/issues/6885
But, according to documentation `defaultViewport` can be null

Reply:
can you add a comment with a description after, e.g. 
```suggestion
        // @ts-ignore ToDo(@L0tso): remove once https://github.com/puppeteer/puppeteer/pull/6942 is merged
```

Reply:
added